src/web/assets/matrix/src/MatrixInput.js

```diff
@@ -241,69 +232,78 @@ import $ from 'jquery';
           );
         }
 
-        const {data} = await Craft.sendActionRequest(
-          'POST',
-          'matrix/create-entry',
-          {
-            data: {
-              fieldId: this.settings.fieldId,
-              entryTypeId: this.entryTypesByHandle[type].id,
-              ownerId: this.settings.ownerId,
-              ownerElementType: this.settings.ownerElementType,
-              siteId: this.settings.siteId,
-              namespace: this.settings.namespace,
-              staticEntries: this.settings.staticEntries,
-            },
+        this.elementEditor?.queue.push(async () => {
```

Comment:
@brianjhanson This would prevent the entry from getting created if `elementEditor` isn’t set. Would going with `Craft.queue.push(` work instead?

Reply:
@brandonkelly yep, seems to work just as well. Thank you!